OrbitLinuxTracing/PerfEventRecords.h

```diff
@@ -86,6 +86,17 @@ struct __attribute__((__packed__)) perf_event_empty_sample {
   perf_event_sample_id_tid_time_streamid_cpu sample_id;
 };
 
+// This struct must be in sync with the SAMPLE_REGS_URET_PROBE in
+// PerfEventOpen.h.
+struct __attribute__((__packed__)) perf_event_uretprobe {
```

Comment:
Similarly, this should be called `perf_event_ax_sp_ip`, for consistency with the other structs: the names carry the content, not the usage.